src/ansys/fluent/core/fluent_connection.py

```diff
@@ -285,7 +285,7 @@ def check_health(self) -> str:
 
     def get_fluent_version(self):
         """Gets and returns the fluent version."""
-        return ".".join(map(str, self.scheme_eval.scheme_eval("(cx-version)")))
+        return self.version
```

Comment:
Looks like this is now going to return '(23 2 0)' rather than '23.2.0'. To rectify this, see above.

Reply:
Yes, we have closed this PR now.